src/Extension/Task/DeprecationsCheckTask.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 id: "validation:php:deprecations-check"
-short_description: "Check your code by running a static code analysis and generate report with deprecations"
-help: ~
+short_description: "Check your code by running a static code analysis and generate report with deprecations. By default uses 'psalm.xml' config file."
```

Comment:
` is usually used for code fencing. Any reason this cannot be done here?

Reply:
sure